xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.js

```diff
@@ -441,7 +441,13 @@ XWiki.widgets.LiveTable = Class.create({
         }
         // The value can be passed as a string..
         if (descriptor.html + '' === 'true') {
-          container.innerHTML = row[fieldName] || '';
+          // The value of the column must be unescaped to allow it to be used as html.
```

Comment:
Could you explain why it's escaped? Is it JSON-escaped? If so that's true for any value not because it's used as HTM. What unescaping are you referring to? I don't see any unescape call below in your changes.